src/makeRobot.js

```diff
@@ -38,7 +38,73 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const initialCoordinates = 0;
+  const evacuateCoordinateX = 1400;
+  const evacuateCoordinateY = 500;
```

Comment:
looks clearer this way, I believe
```suggestion
  const initialCoordinates = 0;
  const initialCoordinates = {
    x: 0,
    y: 0,
  }
  
  const evacuationCoordinates = {
    x: 1400,
    y: 500,
  }:
```